charts/testkube-operator/templates/deployment.yaml

```diff
@@ -40,7 +40,6 @@ spec:
         args:
         - --secure-listen-address=0.0.0.0:8443
         - --upstream=http://127.0.0.1:8080/
-        - --logtostderr=true
```

Comment:
why it was removed?

Reply:
DEPRECATED: will be removed in a future release, see https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/2845-deprecate-klog-specific-flags-in-k8s-components

Instead of deleting it, we can add a choice whether to use this option or not.

Reply:
@batazor can we make this optional for now?